core/templates/base-components/loading-message.component.ts

```diff
@@ -24,7 +24,9 @@ import { downgradeComponent } from '@angular/upgrade/static';
    templateUrl: './loading-message.component.html'
  })
 export class LoadingMessageComponent {
-   @Input() message: string;
+  // Angular lifecycle hooks are used to populate values, see
+  // https://github.com/oppia/oppia/wiki/Guide-on-defining-types.
```

Comment:
Should we rather mention point number here like [TS-7-1] along with the link?

Reply:
@vojtechjelinek recommended putting up a small description alongside the link of the wiki. When the wiki is up, I think we can put the link to the specific section and then there won't be a need to specify the case number explicitly.

Reply:
Yes the exact link to that particular case would be fine!

Reply:
done